pkg/server/status.go

```diff
@@ -2889,13 +2889,23 @@ func (s *systemStatusServer) HotRangesV2(
 		ErrorsByNodeID: make(map[roachpb.NodeID]string),
 	}
 
-	var requestedNodes []roachpb.NodeID
-	if len(req.NodeID) > 0 {
-		requestedNodeID, local, err := s.parseNodeID(req.NodeID)
+	nodes := req.Nodes
+	if req.NodeID != "" {
+		nodes = append(nodes, req.NodeID)
```

Comment:
nit: no need to change but to me, this is confusing behavior. we should either use `NodeID` or `Nodes`, not both.

Reply:
This is to support backwards compatibility -- if there's a mixed version cluster, not handling both in the new version can result in the cluster crashing.